tests/test_taskgroups.py

```diff
@@ -925,3 +973,12 @@ async def taskfunc(*, task_status):
             tg.start_soon(sleep, 5)
             await wait_all_tasks_blocked()
             task.cancel('blah')
+
+
+async def test_raised():
+    with pytest.raises(ValueError):
```

Comment:
What does this test test for that wouldn't be caught by previous tests?

Reply:
Removed